man/systemd.network.xml

```diff
@@ -1130,6 +1130,39 @@
         </varlistentry>
       </variablelist>
   </refsect1>
+  <refsect1>
+    <title>[BridgeVLAN] Section Options</title>
+      <para>The <literal>[BridgeVLAN]</literal> section manages the
+      VLAN ID configuration of a bridge port and accepts the following
+      keys. Specify several <literal>[BridgeVLAN]</literal> sections to
+      configure several VLAN entries. The VLANFiltering= option has to be
```

Comment:
Enclose `VLANFiltering=` in `<varname>` please. Also, please mention that VLANFiltering= is found in the .netdev files [Bridge] section, and add a link to the man page for that, please